### ISSUE TITLE
UI/UX bug with alertbar

### ISSUE THREAD

**Issue by o1dnik** [telegramdesktop/tdesktop#6065]
### Steps to reproduce
1. Start a call with another user
2. Delete some other channel or chat
3. New alert should appear with "undo" button for 3-5 sec
4. After the "undo" alertbar disappears - to see a current call blue bar "End call" have to open another chant to trigger the alertbar view

### Expected behaviour
After "undo" alert disappears - the current call alertbar must become visible again

### Actual behaviour
After "undo" alert disappears - the current call alertbar is hidden

### Configuration
MAC OS Mojave v10.14.5

**Version of Telegram Desktop:**
5.2.2 (171178) Stable

**Used theme**:
Standard

<details><summary>No Logs</summary>
</details>

**Comment by 23rd** [comment]
Sorry, you are using the Mac native client, not the Telegram Desktop.

**Comment by o1dnik** [comment]
em... thanks @23rd , but what's the difference? Isn't Telegram Desktop App for Macintosh is build from some this source? It's just a different app which uses the same Telegram servers APIs ?

**Comment by 23rd** [comment]
Here is a comment that will explain better.
https://github.com/telegramdesktop/tdesktop/issues/5921#issuecomment-483126605

**Comment by o1dnik** [comment]
Okay, thanks for the comment. I will post it on their repo. But I would allow myself to leave a link on a comment for other people who might be interested in clarifications of differences in applications if you don't mind.

I found a very short explanation in [official Telegram FAQ](https://telegram.org/faq#q-why-do-you-have-two-apps-in-the-mac-app-store) which I somehow missed. But you know, I installed both now and will use both for a while. Already see several differences in UI and there are of course a bit more in functionality for sure. I also found [this](http://tsfkb.wikidot.com/tips:tdesktop:different-versions) short explanation on the Desktop wiki a little biased towards Telegram Desktop. Also the statement by user lloeki on HackerNews that the Desktop app is based on Electron is incorrect. It's not 😄 

I wonder though about two things now: 1) applications started more or less in parallel, but one is under GPLv2 and another is underGPLv3. What reason would be for this? I mean I probably understand that it's not BSD just to restrict possible incorporation into proprietary software, to keep it open-sourced. But GPLv3 seems to have more of those restrictions compared to v2. I found the speech of Linus Torwalds on another DebConf ([video](https://telegram.org/faq#q-why-do-you-have-two-apps-in-the-mac-app-store)) very useful regarding this.
And the second - 2) it seems that the native one for MAC OS is either more stable or less volatile in changes according to the contributions, but I can't clearly answer this second question. It's usually harder to support a cross-platform app development compared to an OS-targeted development.

Sorry for long text, I can edit or cut it if you want to keep thread clean though interested in direct opinion of a maintainer.
